routes/merged/project.rb

```diff
@@ -79,13 +79,8 @@ class Clover
 
         @project.soft_delete
 
-        if api?
-          response.status = 204
-          r.halt
-        else
-          flash["notice"] = "'#{@project.name}' project is deleted."
-          return {message: "'#{@project.name}' project is deleted."}
```

Comment:
I assume this request is submitted via javascript if we are returning 204 with an empty body. Does the UI still show an appropriate message in this case?

Reply:
Yes, 204 is success for an ajax request

https://github.com/ubicloud/ubicloud/blob/ce499b0ff3f719d0cecc8ca65bff33171b88f3b3/assets/js/app.js#L79